cpp/lib/include/BSZTransform.h

```diff
@@ -0,0 +1,65 @@
+/*  \brief Utility Functions for Transforming Data in accordance with BSZ Requirements
+ *  \copyright 2018 Universitätsbibliothek Tübingen.  All rights reserved.
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "MARC.h"
+#include "Zotero.h"
+
+namespace Zotero { struct SiteParams; } // forward declaration
```

Comment:
why necessary? is part of Zotero.h

Reply:
Necessary to resolve include dependency problems.